docs/user/rstudio/readme.md

```diff
@@ -0,0 +1,46 @@
+# RStudio User Guide
+
+-   Source code: <https://github.com/rstudio/rstudio/tree/main/docs/user/rstudio>
+-   Public URL: <https://docs.rstudio.com/ide/user/>
+
+> If you’re interested in getting involved with open source, contributing to documentation is a good—and helpful—place to start. - [James Turnbill](https://increment.com/documentation/documentation-as-a-gateway-to-open-source/)
+
+If you have found issues with the RStudio User Guide, please check for existing duplicate issues/pull requests. 
+If a matching Issue/Pull Request doesn't exist, please then open an [Issue](https://github.com/rstudio/rstudio/issues/new/choose) or Pull Request. 
+Each page in the RStudio User Guide docs have an option to view the source which is useful for specifying a line of code in an Issue. 
+Alternatively, selecting "Edit this Page" will prepare you to open a quick Pull Request from your browser.
+
+## Development
+
+To locally test changes to the User Guide, you will need to do the following:
+
+1.  [Install Quarto](https://quarto.org/docs/getting-started/installation.html) or RStudio 2022.07.2 or later, which bundles Quarto.
+
+-   Download RStudio from [posit.co/downloads/](https://posit.co/downloads/)
+-   Download the CLI from [github.com/quarto-dev](https://github.com/quarto-dev/quarto-cli/releases/latest) for your operating system
```

Comment:
What's the reason for including this link when we've given instructions for downloading the CLI in line 17, `1. [Install Quarto]` etc.?